docker/docker-compose.yaml

```diff
@@ -137,7 +137,6 @@ services:
         condition: service_started
     volumes:
       - ./data/zkevm/sequencer.keystore:/pk/sequencer.keystore
-      - ./data/zkevm/aggregator.keystore:/pk/aggregator.keystore
```

Comment:
Should we revert this perhaps? From what I can see, we are starting the aggregator component (https://github.com/0xPolygon/agglayer/pull/90/files#diff-89fafc265c1fa601cf2364b20be83308031335fda9467fcf3249d5ec1c0c8172R123). There might be some error logging in the node logs, but it is not a critical one I'd say.